distributions/aws/examples/rds-s3/README.md

```diff
@@ -105,23 +107,23 @@ Follow this [doc](https://www.kubeflow.org/docs/distributions/aws/customizing-aw
       ```
 
 
-### 4. Install using the following command:
+### 4. Build manifests and Install Kubeflow
```

Comment:
should manifests be capitalized here? I see it capitalized earlier when referring to the Manifests working group